main.js

```diff
@@ -0,0 +1,90 @@
+const mobileMenuOverlay = document.querySelector('.mobileMenu');
+const openMenuButton = document.querySelector('.mobileMenuIcon');
+const closeMenuButton = document.querySelector('.closeMenuIcon');
+
+function toggleMenu() {
+  mobileMenuOverlay.style.display = 'block';
+
+  closeMenuButton.style.display = 'block';
+  openMenuButton.style.display = 'none';
+}
+
+function closeMenu() {
+  mobileMenuOverlay.style.display = 'none';
+  openMenuButton.style.display = 'block';
+  closeMenuButton.style.display = 'none';
+}
+
+openMenuButton.addEventListener('click', () => toggleMenu());
+closeMenuButton.addEventListener('click', () => closeMenu());
+
+const speakerCard = document.querySelector('.SpeakerSection');
+
+const speakersinfo = [
+  {
+    Name: 'Mwiya Musokotwane',
+    Picture: './images/Mwiya1.jpg',
+    ImageAlt: 'Mwiya Musokotwane',
+    speakerTitle: 'CEO Thebe Investment',
+    Bio: 'Mwiya Musokotwane is the CEO of Zambia-based investment firm Thebe Investment Management.',
```

Comment:
![image](https://user-images.githubusercontent.com/87689547/153265493-6adf7c1f-1b83-4113-a213-147a93ad63a8.png)
- [ ] I noticed images are working in our local but it fails when deployed, Let's make sure we fix that.

Reply:
Fixed,thanks